### DISCUSSION TITLE
Wear all feature suggestion

### DISCUSSION THREAD

**Discussion by ArmoredArtist** [GoMudEngine/GoMud#90] [Ideas]
You mentioned to me that you have features for wear all, remove all for your inventory. It would be interesting to have an outfit command. You could put on what you want and then set the "outfit". Then you could remove all and wear just that outfit any time you wanted to instantly change the look of your character's wardrobe without manually finding it all.

**Comment by Volte6** [comment]
That's a really cool idea actually, so basically a preset of objects that you can store/recall? Of course, you'd then be carrying all those objects around with you at all times in order to do a costume change... hmmm....

This also leads into another idea I've been mulling which is `item sets` - ensembles that when all worn together yield an additional benefit. I haven't fully thought it through yet, but it's something I like.

**Reply by ArmoredArtist** [reply]
You could have a changing area that the outfit command works in that you store your clothes. Realistically nobody would carry all the clothes they are not wearing anyway so it would already be somewhere else stored.

**Reply by ArmoredArtist** [reply]
It might work better if you had a feature to look for the items in the "outfit" in your vicinity and equip what items it could find from the "outfit set" that way it would not have to be at a specific location but work dynamically wherever you might store items. If you could figure out how to make it look in all available containers your in front of.

**Comment by Volte6** [comment]
I'm going to close this topic as it's not clear how I would approach it yet, or if I want to - right now I think item sets for the purposes of special bonuses/effects is the direction I'll go
